static/carousel/js/carousel.js

```diff
@@ -83,7 +121,7 @@
                 });
 
                 $carousel.slick({
-                    slidesToShow: 3,
+                    slidesToShow: config.no_of_videos,
```

Comment:
Ok, please declare this on  `defaults/carousel.json` as well. you will see other options in that  file that will help you as example